apache-rat-plugin/pom.xml

```diff
@@ -160,7 +160,7 @@
             <configuration>
               <includeArtifactIds>apache-rat-core</includeArtifactIds>
               <excludeClassifiers>tests</excludeClassifiers>
-              <include>META-INF/RELEASE_NOTES.txt</include>
+              <includes>META-INF/RELEASE_NOTES.txt</includes>
```

Comment:
@Claudenw should we add a configuration for maven-clean-plugin as well in order to get rid of the generated files?